snippets/buy-buttons.liquid

```diff
@@ -79,15 +79,19 @@
             {% endif %}
           >
             <span>
-              {%- if product.selected_or_first_available_variant.available == false or quantity_rule_soldout -%}
-                {{ 'products.product.sold_out' | t }}
-              {%- else -%}
-                {{ 'products.product.add_to_cart' | t }}
-              {%- endif -%}
+              {% unless product.metafields.custom.out_of_stock == true %}
```

Comment:
this conditional could be rewritten to be less confusing. The `== true` is unnecessary.